charts/datagovuk/templates/opensearch-sts/service.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-opensearch-sts
+spec:
+  selector:
+    app: {{ .Release.Name }}-opensearch-sts
+  ports:
+    - name: host-port
+      protocol: TCP
+      port: 9200
+      targetPort: host-port
+    - name: node-port
+      protocol: TCP
```

You can declutter this a little cos `protocol: TCP` is the default
```suggestion
      port: 9200
      targetPort: host-port
    - name: node-port
```